ddtrace/propagation/http.py

```diff
@@ -1152,6 +1152,16 @@ def my_controller(url, headers):
                         context._baggage = baggage_context.get_all_baggage_items()
                     else:
                         context = baggage_context
+                    if config._baggage_tag_keys:
+                        for baggage_tag_key in config._baggage_tag_keys:
+                            if (stripped_key := baggage_tag_key.strip()) and (
+                                (value := baggage_context.get_baggage_item(stripped_key)) is not None
+                            ):
+                                prefixed_key = "baggage." + stripped_key
```

Comment:
```suggestion
                                prefixed_key = BAGGAGE_TAG_PREFIX + stripped_key
```
nit but maybe would be good to make "baggage." a constant under a name like BAGGAGE_TAG_PREFIX or something like that.